src/backend/main/py.main/caliopen_main/user/store/user.py

```diff
@@ -74,11 +74,12 @@ class RemoteIdentity(BaseModel):
     """User remote identities model."""
 
     user_id = columns.UUID(primary_key=True)
-    identity_id = columns.Text(primary_key=True)
+    identifier = columns.Text(primary_key=True)
+    display_name = columns.Text()
     type = columns.Text()
     status = columns.Text()
-    credentials = columns.List(columns.Text)
     last_check = columns.DateTime()
+    infos = columns.Map(columns.Text, columns.Text)
```

Comment:
Is this field for storing credentials ? Or do we need to add 2 more fields : `username`, `password`.  
BTW, we need to choose a more secure storage strategy for remote identities credentials.